repositories/user.go

```diff
@@ -85,12 +86,12 @@ func (r *UserRepository) GetByLoggedInAfter(t time.Time) ([]*models.User, error)
 // NOTE: Only ID field will be populated
 func (r *UserRepository) GetByLastActiveAfter(t time.Time) ([]*models.User, error) {
 	subQuery1 := r.db.Model(&models.Heartbeat{}).
-		Select("user_id as user, max(time) as time").
+		Select("user_id , max(time) as time").
 		Group("user_id")
 
 	var userIds []string
 	if err := r.db.
-		Select("user as id").
+		Select("user").
 		Table("(?) as q", subQuery1).
 		Where("time >= ?", t.Local()).
 		Scan(&userIds).Error; err != nil {
```

Comment:
These changes break the `GetByIds` function. Getting a 500 now when request, for example, the `/api/metrics` endpoint with SQLite:

```
2024-01-12T15:29:56.148722234+01:00 [ERROR] failed to retrieve active users for metric - SQL logic error: no such column: user (1)
2024-01-12T15:29:56.148733462+01:00 [ERROR] SQL logic error: no such column: user (1)
```

Reply:
Yes. This is because the `user` is `select`ed from the subquery above. The `user_id` column was previously given a alias `user`. But the alias was wrongly removed. I am fixing it in the next commit.